### DISCUSSION TITLE
Ошибка

### DISCUSSION THREAD

**Discussion by VitaliyDenysiuk** [community/community#32902] [Other Feature Feedback, Questions, & Ideas]
Вылетает ошибка во время Пуша на GitHub, почитал  возможные причины, скорее всего приложение весит больше чем допускается. Как можно расширить GitHub что бы пушить приложения которые весят больше допускаемого?

**Comment by jfullstackdev** [comment]
as much as we want to help, but you can try translating it first using an online translator
